### DISCUSSION TITLE
Arti dari encoding

### DISCUSSION THREAD

**Discussion by abbymakarim** [teamalgoritma/community#20] [Q&A] [answered]
<img width="1054" alt="Screen Shot 2022-03-18 at 20 30 22" src="https://user-images.githubusercontent.com/62086533/159011769-f22bec49-c0ad-4998-b741-2a6de93b3179.png">

untuk penjelasan mengenai encoding bagaimana ya?

**Comment by dwiknrd** [comment, marked as answer]
Ketika melakukan read data menggunakan fungsi `read_csv` ada parameter `encoding` yang dapat kita atur yang bertujuan untuk mengubah suatu format data menjadi bentuk/format yang bisa dibaca. Ada beberapa encoding yang bisa bapak gunakan untuk membaca / menulis yang digunakan oleh Python yaitu **UTF-8**.

`unicode_escape` sendiri akan mengonversi satu karakter ke format titik kode heksadesimal 4 digit, seperti \uXXXX.
